src/config/testnet_postake_medium_curves.mlh

```diff
@@ -1,4 +1,4 @@
-[%%define ledger_depth 14]
+[%%define ledger_depth 17]
```

Comment:
Is this for testing only, or is this a change that we want to land in develop?

Reply:
good question -- we do want to change this at some point as we would like to support at least 100k accounts, and things seem to work on QA nets with this ledger depth. The only open question is what the performance of bootstrap will be like -- it wasn't easy to immediately test since we would have to have a node who has a ledger which is significantly different from the genesis ledger, or maybe some hacked in thing to test the same thing.

In any case, it has nothing to do with the rest of my diff so I will get rid of the change.

Reply:
Depending on what we do for the hard-fork planning, that might be a good way to test this easily.